static/js/publisher/metrics/graphs/activeDevices.js

```diff
@@ -147,12 +205,23 @@ export default function activeDevices(days, activeDevices) {
         group
       ],
       x: 'x',
-      columns: [days].concat(activeDevices)
+      columns: [days].concat(deviceData)
+    },
+    onrendered: () => {
+      // Because we're modifying the axis with CSS the mask crops the labels
+      // We don't need the masks, so here we remove them each render.
+      Array.from(
+        el.querySelectorAll('clipPath')
+      ).forEach(mask => {
+        if (mask.id.indexOf('axis') !== -1) {
+          mask.remove();
```

Comment:
I guess it would be good to define browsers we support. `remove` is not available in IE at all (only in Edge). So are many other new features (arrow functions, const/let, etc) that we use and don't transpile.

Especially that we develop and test only in latest browsers where all of this works, and it will just throw badly when just one feature doesn't work.

Not sure if it's something to fix in this PR, but something we may want to address now that we add more and more JS features.

Reply:
Yeah good point, I've been thinking we should add some transpilation soon, even just to figure out how we want to do it moving forward, and it should help with things like this.